packages/data/src/registry.js

```diff
@@ -140,7 +140,7 @@ export function createRegistry( storeConfigs = {} ) {
 	 */
 	function registerReducer( reducerKey, reducer ) {
 		const enhancers = [];
-		if ( window.__REDUX_DEVTOOLS_EXTENSION__ ) {
+		if ( window && window.__REDUX_DEVTOOLS_EXTENSION__ ) {
```

Comment:
I think this check should be `typeof window !== 'undefined' &&` -- otherwise, https://github.com/Automattic/wp-calypso/pull/26838 won't even start for me but fail with a `ReferenceError: window is not defined`.

Reply:
Thanks, will update.